src/hardware/vga_other.cpp

```diff
@@ -616,20 +616,14 @@ static void PCJr_FindMode()
 			} else {
 				VGA_SetMode(M_TANDY2);
 			}
-		} else {
-			/* otherwise some 4-colour graphics mode */
+		}
+		/* otherwise some 4-colour graphics mode */
```

Comment:
Personal comment: I don't like when we break if-else formatting by forcing comments like in here - it results in 2 different styles for formatting code. Can we go back to previous way, or maybe place the comment after `} else {` in the same line?

Reply:
Absolutely; thanks @dreamer .

I was trying to get the if and else comments both aligned. 

`c++
// describe positive branch
if (positive-condition) {
    ....
} else {
    // describe negative branch
    ...
}
``` 

I don't like how the negative condition comment is not aligned with the positive comment - given they are peers.

Should I move the positive-condition comments inside the `if` too?